tools/wptrunner/wptrunner/browsers/chrome.py

```diff
@@ -40,10 +43,14 @@ def browser_kwargs(logger, test_type, run_info_data, config, **kwargs):
 
 def executor_kwargs(logger, test_type, test_environment, run_info_data,
                     **kwargs):
+    sanitizer_enabled = kwargs.get("sanitizer_enabled")
+    if sanitizer_enabled:
```

Comment:
Moved my previous comment here.

Seems like you are trying to generalize this to make it work for both chrome and content shell, and looks it will work except some minor things, e.g. at line 70 and 134 a different browser side config will be used due to the change to the test type.

Another question is how will the log be saved. For content shell there will be sanitizer log from stderr. What is the case for chrome?